src/components/LayerCollection/LayerPOLine.js

```diff
@@ -233,8 +238,11 @@ class LayerPOLine extends Component {
     } = this.props;
     const { layer } = location.search ? queryString.parse(location.search) : {};
     const order = this.getOrder();
+    const vendor = order
+      ? get(parentResources, 'vendors.records', []).find(d => d.id === order.vendor)
```

Comment:
I don't see any point in this condition, could you please provide one if you have?

Reply:
We use order.vendor as key for search and until order is undefined we get error

Reply:
seems you need to `get` `id` before getting vendor to not have condition